integration-tests/src/lib.rs

```diff
@@ -111,7 +111,7 @@ impl Tester {
         let l2_rpc_address = format!("0.0.0.0:{}", l2_locked_port.port);
         let l2_rpc_ws_url = format!("ws://localhost:{}", l2_locked_port.port);
         let prover_api_address = format!("0.0.0.0:{}", prover_api_locked_port.port);
-        let prover_api_url = format!("http://localhost:{}", prover_api_locked_port.port);
+        let _prover_api_url = format!("http://localhost:{}", prover_api_locked_port.port);
```

Comment:
You can just move this logic inside `if enable_prover` below so it doesn't get marked as `unused` for non-prover tests